docker/conf/env/create-env-linux-1.sh

```diff
@@ -38,6 +38,9 @@ export KC_JEMPI_CLIENT_SECRET="Tbe3llP5OJIlqUjz7K1wPp8YDAdCOEMn"
 export KC_JEMPI_ROOT_URL="http://localhost:3000"
 export JEMPI_SESSION_SECRET="c05ll3lesrinf39t7mc5h6un6r0c69lgfno69dsak3vabeqamouq4328cuaekros401ajdpkh60rrt"
 export JEMPI_FILE_IMPORT_MAX_SIZE_BYTE=128000000
+# Deployment related env vars
+export JEMPI_SESSION_SECURE=false
+export DOMAIN_NAME=none
```

Comment:
shouldn't this be localhost by default ?

Reply:
okay cool